projects/plugins/jetpack/extensions/blocks/ai-assistant/inline-extensions/with-ai-extension.tsx

```diff
@@ -428,8 +429,16 @@ const blockEditWithAiComponents = createHigherOrderComponent( BlockEdit => {
 			};
 		}, [ adjustBlockPadding, clientId, controlObserver, id, showAiControl ] );
 
+		const InlineExtensionsProvider = childrenBlock
+			? React.Fragment
+			: InlineExtensionsContext.Provider;
+
+		const ProviderProps = childrenBlock
+			? {}
```

Comment:
```suggestion
			? { value: {} }
```

To fix the typescript error.